big_tests/tests/service_domain_db_SUITE.erl

```diff
@@ -412,6 +434,116 @@ db_out_of_sync_crashes_node(_) ->
     true = rpc(mim(), meck, num_calls, [mongoose_domain_utils, halt_node, 1]) > 0,
     ok.
 
+cli_can_insert_domain(Config) ->
+    {"Added\n", 0} =
+        ejabberdctl("insert_domain", [<<"example.db">>, <<"type1">>], Config),
+    {ok, #{host_type := <<"type1">>, enabled := true}} =
+        select_domain(mim(), <<"example.db">>).
+
+cli_can_disable_domain(Config) ->
+    ejabberdctl("insert_domain", [<<"example.db">>, <<"type1">>], Config),
+    ejabberdctl("disable_domain", [<<"example.db">>], Config),
+    {ok, #{host_type := <<"type1">>, enabled := false}} =
+        select_domain(mim(), <<"example.db">>).
+
+cli_can_enable_domain(Config) ->
+    ejabberdctl("insert_domain", [<<"example.db">>, <<"type1">>], Config),
+    ejabberdctl("disable_domain", [<<"example.db">>], Config),
+    ejabberdctl("enable_domain", [<<"example.db">>], Config),
+    {ok, #{host_type := <<"type1">>, enabled := true}} =
+        select_domain(mim(), <<"example.db">>).
+
+cli_can_delete_domain(Config) ->
+    ejabberdctl("insert_domain", [<<"example.db">>, <<"type1">>], Config),
+    ejabberdctl("delete_domain", [<<"example.db">>, <<"type1">>], Config),
+    {error, not_found} = select_domain(mim(), <<"example.db">>).
+
+cli_cannot_delete_domain_without_correct_type(Config) ->
+    {"Added\n", 0} =
+        ejabberdctl("insert_domain", [<<"example.db">>, <<"type1">>], Config),
+    {"Error: \"wrong host type\"\n", 1} =
+        ejabberdctl("delete_domain", [<<"example.db">>, <<"type2">>], Config),
+    {ok, _} = select_domain(mim(), <<"example.db">>).
+
+cli_cannot_insert_domain_twice_with_the_another_host_type(Config) ->
```

Comment:
very minor: 'the another' looks awkward - it's a duplicate determiner, 'another' would suffice